Use 'rotateRadians' in CanvasRenderingContext2DBase::rotate

Ahmad-S792 opened the pull request:
#### 1df0b1914071a4a3f3f4543620ac7d4daf38e6b1
<pre>
Use &apos;rotateRadians&apos; in CanvasRenderingContext2DBase::rotate

<a href="https://bugs.webkit.org/show_bug.cgi?id=265149">https://bugs.webkit.org/show_bug.cgi?id=265149</a>

Reviewed by Chris Dumez and Tim Nguyen.

In 268964@main, we added helper function &apos;rotateRadians&apos; to reduce unnecessary operations, this patch is
to extend the usage of this helper in `rotate` function in &apos;CanvasRenderingContext2DBase.cpp&apos;.

* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
(CanvasRenderingContext2DBase::rotate):

Canonical link: <a href="https://commits.webkit.org/270994@main">https://commits.webkit.org/270994@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dc3c44fa5f22253e571d139267d26ab0b31157b1

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/27020 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/5636 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/28256 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/29229 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/24705 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/27473 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/7500 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/3028 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/24565 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/27282 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/4456 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/23193 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/3898 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/3978 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/24191 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/29865 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/24660 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/24601 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/30183 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/3993 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/2187 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/28111 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/5456 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/6491 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/4459 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/4365 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
You should be able to remove the `#include <wtf/MathExtras.h>`

cdumez reviewed (approved):
LGTM

Ahmad-S792 committed:
Use 'rotateRadians' in CanvasRenderingContext2DBase::rotate

https://bugs.webkit.org/show_bug.cgi?id=265149

Reviewed by Chris Dumez and Tim Nguyen.

In 268964@main, we added helper function 'rotateRadians' to reduce unnecessary operations, this patch is
to extend the usage of this helper in `rotate` function in 'CanvasRenderingContext2DBase.cpp'.

* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
(CanvasRenderingContext2DBase::rotate):

Canonical link: https://commits.webkit.org/270994@main
